projects/packages/classic-theme-helper/webpack.config.js

```diff
@@ -0,0 +1,94 @@
+const path = require( 'path' );
+const jetpackWebpackConfig = require( '@automattic/jetpack-webpack-config/webpack' );
+const glob = require( 'glob' );
+
+const sharedWebpackConfig = {
+	mode: jetpackWebpackConfig.mode,
+	devtool: jetpackWebpackConfig.devtool,
+	output: {
+		...jetpackWebpackConfig.output,
+		path: path.resolve( './build' ),
+	},
+	optimization: {
+		...jetpackWebpackConfig.optimization,
+	},
+	resolve: {
+		...jetpackWebpackConfig.resolve,
+		modules: [ 'node_modules' ],
+		alias: {
+			...jetpackWebpackConfig.resolve.alias,
+			fs: false,
+		},
+	},
+	node: {},
+	externals: {
+		...jetpackWebpackConfig.externals,
+		jetpackConfig: JSON.stringify( {
+			consumer_slug: 'jetpack-classic-theme-helper',
+		} ),
+	},
+	module: {
+		strictExportPresence: true,
+		rules: [
+			// Transpile JavaScript
+			jetpackWebpackConfig.TranspileRule( {
+				exclude: /node_modules\//,
+			} ),
+
+			// Handle CSS.
+			jetpackWebpackConfig.CssRule( {
+				extensions: [ 'css', 'sass', 'scss' ],
+				extraLoaders: [
+					{
+						loader: 'postcss-loader',
+						options: {
+							postcssOptions: { plugins: [ require( 'autoprefixer' ) ] },
+						},
+					},
+					{
+						loader: 'sass-loader',
+						options: {
+							sassOptions: {
+								// The minifier will minify if necessary.
+								outputStyle: 'expanded',
+							},
+						},
+					},
+				],
+			} ),
```

Comment:
I think this could be simplified to just the following, as we don't need to compile sass (and can remove the related composer devDependencies):
```suggestion
			jetpackWebpackConfig.CssRule( {
				extensions: [ 'css' ],
			} ),
```